### DISCUSSION TITLE
2024-02-13 rust-libp2p Open Maintainers Call

### DISCUSSION THREAD

**Discussion by dhuseby** [libp2p/rust-libp2p#5150] [General]
Hello!! As part of cleaning up the [libp2p Community Calendar](https://lu.ma/libp2p) I took the liberty of rescheduling the rust-libp2p Open Maintainers Call on the same schedule as before only later in the day by a few hours. The next call will take place on February 13th. The event page is here: [2024-02-13 rust-libp2p Open Maintainers Call](https://lu.ma/rust-libp2p). I've already created a meeting doc in the shared [libp2p meetings notes folder](https://drive.google.com/drive/folders/1IifxEdoKEV9ctlLn30I5JBnCg0fJHO7t?usp=sharing). Please feel free to add in any issues you'd like to triage. I've already added some community business to announce/discuss.

In general, the rust-libp2p project is being maintained by volunteers and I'd like to formalize that by refreshing the list of the official maintainers of this project. Feel free to reply here and I'll compile a list and update the permissions.

I hope to see all of you at the maintainers call. @winksaville @shamil-gadelshin @melekes @rkuhn @MarcoPolo @mxinden @thomaseizinger @p-shahi @tomaka @romanb @twittner @elenaf9 @jxs @umgefahren @tcoratger @eira-fransham @AgeManning @dvc94ch @dariusc93 @leonzchang @oblique @DougAnderson444 @vnermolaev @b-zee 

Cheers! 🍻

**Comment by AgeManning** [comment]
I'm interested in being added to the list. We still help look after the gossipsub implementation

**Reply by dhuseby** [reply]
@AgeManning excellent! Let's talk at the maintainers' call next week about who should be added to the list of maintainers.

**Comment by thomaseizinger** [comment]
As I commented on https://github.com/libp2p/rust-libp2p/discussions/5154, the new time is 2am for AEDT so I won't be able to join this.

**Reply by dhuseby** [reply]
It seems like if @thomaseizinger is to attend in the future the only small window where everybody can attend is 14:00 UTC-7, which is 08:00 UTC+11 for you in AUS and 22:00 UTC+1 for our friends in Europe.

How does everybody feel about rescheduling the meeting to this new time slot?

**Reply by thomaseizinger** [reply]
You don't have to reschedule it for me specifically. I can't commit to actually being there I am afraid. @AgeManning is in the same timezone usually so that might be something to consider!

**Reply by DougAnderson444** [reply]
Sorry I couldn't make it today, had a conflict in the timing. I'll keep an eye out for future meetups as they happen
